app/diagnostics.go

```diff
@@ -326,6 +327,8 @@ func (a *App) trackConfig() {
 	a.SendDiagnostic(TRACK_CONFIG_FILE, map[string]interface{}{
 		"enable_public_links":     cfg.FileSettings.EnablePublicLink,
 		"driver_name":             *cfg.FileSettings.DriverName,
+		"isdefault_directory":     isDefault(cfg.FileSettings.Directory, model.FILE_SETTINGS_DEFAULT_DIRECTORY),
+		"isrelative_directory":    strings.HasPrefix(cfg.FileSettings.Directory, "./"),
```

Comment:
Is checking for `./` sufficient to detect a relative directory? I tried looking through the `LocalFileBackend` code and it seems like even a path such as `data` would still be relative. Perhaps we should report `isabsolute_directory` and check using [filepath.IsAbs](https://golang.org/pkg/path/filepath/#IsAbs) instead?

Reply:
Ha, it's basically the opposite of what I'm doing. It's still more self-explanatory though, so I'll switch it

https://golang.org/src/path/filepath/path_unix.go?s=313:341#L2